VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/models/participant-status-base.ts

```diff
@@ -0,0 +1,37 @@
+import { HostListener } from '@angular/core';
+import { ParticipantStatusUpdateService } from 'src/app/services/participant-status-update.service';
+import { EventType } from 'src/app/services/clients/api-client';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { ActivatedRoute } from '@angular/router';
+
+export abstract class ParticipantStatusBase {
+    constructor(
+        protected participantStatusUpdateService: ParticipantStatusUpdateService,
+        protected logger: Logger,
+        protected route: ActivatedRoute
+    ) {}
+
+    raiseNotSignedIn() {
+        try {
+            const eventConferenceId = this.route.snapshot.paramMap.get('conferenceId');
```

Comment:
Each of the participant classes retrieve the conferenceId upon entry.
There is no need to handle this on exit. It's a safe assumption to expect this line to pass.

I suggest an abstract variable for each of the child classes to inherit is the safest option

Reply:
Done